dist/nipplejs.js

```diff
@@ -320,6 +320,7 @@ function Nipple (collection, options) {
         fadeTime: 250,
         dataOnly: false,
         restOpacity: 0.5,
+        resetNob:true,
```

Comment:
Please, use something that goes with the already used semantic.
Something like `restJoystick` would fit better I think.